lib/rubocop/cop/rails/delegate_allow_blank.rb

```diff
@@ -0,0 +1,59 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop looks for delegations that pass :allow_blank as an option
+      # instead of :allow_nil. :allow_blank is not a valid option to pass
+      # to ActiveSupport's :delegate method.
```

Comment:
I'd use the standard Ruby convention for referring to a method name here, instead of `:delegate` - this looks like a symbol to me.